src/ErrorHandler.php

```diff
@@ -94,25 +113,23 @@ public function handleThrowable(\Throwable $t): void
 
     /**
      * Register this error handler.
+     *
+     * @param bool $exposeDetails
      */
-    public function register(): void
+    public function register(bool $exposeDetails = false): void
```

Comment:
You can use ErrorHandler via `ErrorCatcher` only without registering it as PHP handler via calling `register()`. In this case there's no way to set `exposeDetails`.

Reply:
I think there should be a dedicated method for it as it was before and `register()` should not have a side effect of changing `expose` setting.